oriedita/src/main/java/oriedita/editor/drawing/Grid.java

```diff
@@ -340,6 +348,19 @@ public void draw(Graphics g, Camera camera, int p0x_max, int p0y_max, boolean co
             s_tv.set(camera.object2TV(s_ob));
             g.drawLine((int) s_tv.determineAX(), (int) s_tv.determineAY(), (int) s_tv.determineBX(), (int) s_tv.determineBY()); //直線
         }
+        
+        if(drawDiagonalGridlines && gridAngle != -90){
```

Comment:
aah yes, I think I removed that on an earlier version where they were vertical so not useful.

Reply:
![image](https://user-images.githubusercontent.com/33122845/174498271-80b22e57-7893-42cd-9f3b-b1bc9b23cd87.png)
now it is like this, however... this may need some more work on where to start and end drawing lines.... but also this is less of an issue when the entire grid is shown, which is also always the case if grid lines are not 90

Reply:
Might be nice to fix the case where there is a 90 degree grid in within_paper mode. Or for now don't draw in this specific case.

Reply:
redid the drawing! it's a bit more complex for my liking, bc diagonals are drawn from the screen in stead of having 0 as a reference. added comments to make that clear :)